c/polyjuice.h

```diff
@@ -949,25 +1013,20 @@ int handle_message(gw_context_t* ctx,
                    const evmc_message* msg_origin, struct evmc_result* res) {
   static const evmc_address zero_address{0};
 
-  ckb_debug("[handle_message] BEGIN handle_message");
   evmc_message msg = *msg_origin;
   int ret;
 
   bool to_address_exists = false;
   uint32_t to_id = 0;
   uint32_t from_id;
-  // TODO: passing this value from function argument
-  uint8_t from_script_hash[32] = {0};
-  uint8_t to_script_hash[32] = {0};
+
   if (memcmp(zero_address.bytes, msg.destination.bytes, 20) != 0) {
-    ret = ctx->sys_get_script_hash_by_prefix(ctx, msg.destination.bytes, 20, to_script_hash);
-    if (ret == 0) {
+    ret = load_account_id_by_eth_address(ctx, msg.destination.bytes, &to_id);
+    if (ret != 0) {
+      debug_print_int(
```

Comment:
Shouldn't we return error here? Miss the `return ret;`

Reply:
This `if` branch only indicates that `to_address_exists = false`, we should not return ret here.

Scenario example: Call an ERC20 contract to transfer tokens to an eth_address which related Godwoken account was not created.